lisa/sut_orchestrator/hyperv/source.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import PurePath, PureWindowsPath
+from typing import List, Type
+
+from lisa import schema
+from lisa.node import RemoteNode
+from lisa.tools.ls import Ls
+from lisa.tools.remote_copy import RemoteCopy
+from lisa.tools.unzip import Unzip
+from lisa.util import InitializableMixin, subclasses
+from lisa.util.logger import get_logger
+
+from .schema import LocalSourceSchema, SourceFileSchema, SourceSchema
+
+
+class Source(subclasses.BaseClassWithRunbookMixin, InitializableMixin):
+    def __init__(self, runbook: SourceSchema) -> None:
+        super().__init__(runbook=runbook)
+        self._log = get_logger("source", self.__class__.__name__)
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return SourceSchema
+
+    def download(self, server: RemoteNode) -> List[PureWindowsPath]:
```

Comment:
use `PurePath`, in future, we will consolidate source classes.

Reply:
done.